test/e2e-kuttl/quota-forest/11-assert.yaml

```diff
@@ -14,4 +14,4 @@ metadata:
         quota_context: default
         quota_service: default
 status:
-    state: Completed
+    state: Running
```

Comment:
Mmm I feel this should be Completed without any changes OR we set the command for the container to sleep infinity and then set the state to Running

Reply:
I agree, let it be completed and perhaps we should reduce the sleep time to 10 seconds or less.

Reply:
I changed the test behavior as it was failing, I think we are agreeing that quota management works and its the question of what happens to the workload. I think I can change the workload to run to infinity and keep the Kuttl test as it is. build takes a lot of time to complete and i plan do do this in a separate PR